backend/experiment/actions/consent.py

```diff
@@ -5,34 +5,38 @@
 from django_markup.markup import formatter
 from django.core.files import File
 
+from typing import Literal
+
 from .base_action import BaseAction
 
+RenderFormat = Literal["HTML", "MARKDOWN"]
+
 
-def get_render_format(url: str) -> str:
+def get_render_format(url: str) -> RenderFormat:
     """
     Detect markdown file based on file extension
 
     Args:
         url: Url of the consent file
 
     Returns:
-        File format
+        File format of the consent file (HTML or MARKDOWN)
 
     """
     if splitext(url)[1] == ".md":
         return "MARKDOWN"
     return "HTML"
 
 
-def render_html_or_markdown(dry_text: str, render_format: str) -> str:
+def render_html_or_markdown(dry_text: str, render_format: RenderFormat) -> str:
     """render html or markdown
 
     Args:
-        dry_text: contents of a markdown or html file
-        render_format: type of contents, either 'HTML' or 'MARKDOWN'
+        dry_text (str): contents of a markdown or html file
+        render_format (RenderFormat): type of contents, either 'HTML' or 'MARKDOWN'
```

Comment:
Hm, I wonder whether it wouldn't be actually clearer to drop the `RenderFormat` type and use `Literal["HTML", "MARKDOWN"]` instead? More repetitive but also selve-documenting.

Reply:
Ah, I see you're also starting to adher to the WET principle now ;-)